GCTC-NTGC/gc-digital-talent
# Add process number to pool

Adds the 'Process Number' field to the pool model and the view and edit pool pages.

To test:
1. Visit `/en/admin/pools/{poolId}/edit` and confirm it looks ok.
2. Add a process number to the pool and ensure it updates properly.
3. Navigate to `/en/admin/pools/{poolId}` and confirm it displays the new process number for the pool.

Note: This PR adds untranslated strings to an existing page. How do we want to handle this?
